src/main/java/com/ashkan/moviesapi/constants/constants.java

```diff
@@ -0,0 +1,16 @@
+package com.ashkan.moviesapi.constants;
+
+import java.util.Arrays;
+import java.util.List;
+
+public final class constants {
```

Comment:
Class names should start with UpperCase, this should be a **C**onstant class.

Reply:
done.